feat: notification grouping & Inhibition (#381)
flanksource/docs: mission-control/docs/reference/notifications/_inhibition.mdx

<Fields 
  rows={[
    {
      field: 'depth',
      scheme: 'int',
      description: 'Defines how many levels of child or parent resources to traverse.'
    },
    {
      field: 'direction',

>       scheme: '`string`',

Reviewer: ```suggestion
      scheme: '`inoming`|`outgoing`|`both`',
```

      required: true,

>       description: 'Specifies the traversal direction in relation to the "From" resource. Can be "outgoing" (looks for child resources), "incoming" (looks for parent resources), or "all" (considers both).'

Reviewer: ```suggestion
      description: 'Specifies the traversal direction in relation to the `from` resource. `outgoing` looks for child resources and `incoming` looks for parent resources.'
```
